style.css

```diff
@@ -0,0 +1,134 @@
+body {
```

Comment:
:red_circle: Kindly make sure you stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).As you can see your font-size doesn't reflect the Figma design

Your Design
![ourdesign'](https://user-images.githubusercontent.com/26736582/211884049-76f208ae-e214-4ef0-a600-8a64451fb62b.png)

Figma Design
  <p align="center">
      <img src="https://github.com/microverseinc/curriculum-html-css/raw/main/portfolio/images/m1_setup/headline-mobile.png" alt="Headline" />
    </p>